.github/workflows/wheels-test.yaml

```diff
@@ -78,6 +80,10 @@ jobs:
               - { ARCH: 'amd64', PY_VER: '3.10', CUDA_VER: '12.0.1', LINUX_VER: 'ubuntu20.04', gpu: 'v100', driver: 'latest' }
               - { ARCH: 'arm64', PY_VER: '3.9', CUDA_VER: '12.0.1', LINUX_VER: 'ubuntu20.04', gpu: 'a100', driver: 'latest' }
               - { ARCH: 'arm64', PY_VER: '3.10', CUDA_VER: '12.0.1', LINUX_VER: 'ubuntu20.04', gpu: 'a100', driver: 'latest' }
+              - { ARCH: 'amd64', PY_VER: '3.9', CUDA_VER: '12.2.2', LINUX_VER: 'ubuntu18.04', gpu: 'v100', driver: 'latest' }
```

Comment:
CUDA 12.2.2 doesn't list Ubuntu 18.04 in the support matrix.

Compare:
https://docs.nvidia.com/cuda/archive/12.0.1/cuda-installation-guide-linux/index.html#os-support-policy
https://docs.nvidia.com/cuda/archive/12.2.2/cuda-installation-guide-linux/index.html#os-support-policy

This agrees with the Docker image support. (In the past there have been some discrepancies.) There is no `nvidia/cuda` image for CUDA 12.2 on Ubuntu 18.04:
https://hub.docker.com/r/nvidia/cuda/tags?page=1&name=12.0.1-runtime-ubuntu
https://hub.docker.com/r/nvidia/cuda/tags?page=1&name=12.2.2-runtime-ubuntu

I would drop this line.
```suggestion
```

We also probably need to evaluate whether we should be testing Ubuntu 22.04 somewhere in this matrix.

Reply:
Ok yeah that makes sense to me. I ran into that lack of Ubuntu 18.04 images over in https://github.com/rapidsai/ci-imgs/pull/101#discussion_r1432975445 (which is upstream of this PR), but I wasn't sure whether that was intentional. Those OS Support Policy docs answer that for me, thank you!

Reply:
resolving this thread... I dropped this failing ubuntu18.04 build and added a CUDA 12.2 + ubuntu22.04 one (see PR description).